sk-usbhid.c

```diff
@@ -1149,6 +1152,33 @@ check_sign_load_resident_options(struct sk_option **options, char **devicep)
 	return 0;
 }
 
+int
+sk_test_option(const char *test_option)
+{
+	struct sk_usbhid *sk = NULL;
+	int ret = SSH_SK_ERR_GENERAL, internal_uv;
+
+	fido_init(SSH_FIDO_INIT_ARG);
+
+	sk = sk_probe(NULL, NULL, 0, 0);
```

Comment:
I don't think this will do the right thing when multiple devices are present, which is a case we've worked fairly hard to support. Given multiple devices IMO "device supports feature X" is only useful the presence of a key handle that refers to that device.

Maybe a better way of supporting this would be to add another per-flag, say `SSH_SK_UV_PROVIDED` that is set when the token provides user verification internally. We could set this for biometric keys, etc. Would this help?

I.e. putting something like this in `sk_enroll()`

```
if (check_sk_options(sk->dev, "uv", &internal_uv) == 0 && internal_uv != -1) {
        /* user verification handled by token */
        response->flags &= ~SSH_SK_UV_PROVIDED;
}
```

(yes, this looks a lot like the code that I just removed via Corinna's change)